ci/gpu/build.sh

```diff
@@ -108,11 +108,18 @@ conda config --show-sources
 conda list --show-channel-urls
 
 function install_dask {
-    # Install the main version of dask, distributed, and streamz
-    gpuci_logger "Install the main version of dask, distributed, and streamz"
+    # Install the conda-forge or nightly version of dask and distributed
+    gpuci_logger "Install the conda-forge or nightly version of dask and distributed"
     set -x
-    pip install "git+https://github.com/dask/distributed.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
-    pip install "git+https://github.com/dask/dask.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
+    if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
+        gpuci_logger "gpuci_mamba_retry install dask/label/dev::dask --force-reinstall"
+        gpuci_mamba_retry install dask/label/dev::dask --force-reinstall
```

Comment:
Just making sure - with https://github.com/rapidsai/gpuci-build-environment/pull/238, `dask/label/dev` has a higher priority than `conda-forge`, correct? Because of the way I've [relaxed the pinning](https://github.com/dask/distributed/blob/d3f3012dd362fb3a42f823c964fe9a031570b169/continuous_integration/recipes/dask/meta.yaml#L24) on `dask`'s `dask-core` dependency, that package will get picked up from `conda-forge` if this isn't the case.

Maybe a safer way of doing this install would be

```suggestion
        gpuci_logger "gpuci_mamba_retry install -c dask/label/dev dask --force-reinstall"
        gpuci_mamba_retry install -c dask/label/dev dask --force-reinstall
```

to ensure that `dask/label/dev` is always prioritized.

Reply:
It is. Please see PR ( https://github.com/rapidsai/gpuci-build-environment/pull/238 )

Reply:
@charlesbluca Looks like the code-change command is allowing `conda-forge` `dask` and `dask-core` & `distributed` of `dask/label/dev` channel's to co-exist. So I reverted the change which should now pick all the three packages from same label.